src/index.html

```diff
@@ -1,14 +1,38 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta
+    name="viewport"
+    content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+    "
+  >
+  <meta
+    http-equiv="X-UA-Compatible"
+    content="ie=edge"
+  >
   <title>Snake</title>
-  <link rel="stylesheet" href="./style.css">
+  <link
+    rel="stylesheet"
+    href="./style.css"
+  >
 </head>
 <body>
-  <h1>Snake</h1>
+
+<div class="general">
+  <div class="div_num1 div_num_general">1</div>
```

Comment:
сначала основной класс, потом второстепенный
не привязывай классы к названиям тегов
```suggestion
  <div class="block color-1">1</div>
  <div class="block color-2">2</div>
```

Reply:
✔